tests/org.eclipse.reddeer.eclipse.test/src/org/eclipse/reddeer/eclipse/test/wst/server/ui/view/TestServer.java

```diff
@@ -45,3 +58,51 @@ public void modifyModules(IModule[] arg0, IModule[] arg1,
 			IProgressMonitor arg2) throws CoreException {
 	}
 }
+
```

Comment:
Can you please add comment, why this has been added?

Reply:
I have added an comment.